src/pages/workspace/tags/WorkspaceViewTagsPage.tsx

```diff
@@ -276,7 +276,9 @@ function WorkspaceViewTagsPage({route}: WorkspaceViewTagsProps) {
                         customListHeader={getCustomListHeader()}
                         shouldPreventDefaultFocusOnSelectRow={!DeviceCapabilities.canUseTouchScreen()}
                         listHeaderWrapperStyle={[styles.ph9, styles.pv3, styles.pb5]}
-                        onDismissError={(item) => Tag.clearPolicyTagErrors(policyID, item.value)}
+                        onDismissError={() => {
+                            Tag.clearPolicyTagListError(policyID, route.params.orderWeight, 'required');
+                        }}
```

Comment:
Revert. The list here is for tags and calling `clearPolicyTagErrors` is correct.

Follow up?: The `clearPolicyTagErrors` does not support multi level tag i.e. we are not passing the tag list index and it's always using `0`

Reply:
so... I can delete the `clearPolicyTagListError` method from `Tag`?

Reply:
No, we need it for tag lists but not for tags.
- TagsList: Clear error with clearPolicyTagListError
- Tag: Clear error with clearPolicyTagErrors

What you did in your latest commit is correct